src/couch/src/couch_db.erl

```diff
@@ -414,37 +390,34 @@ get_before_doc_update_fun(#db{before_doc_update = Fun}) ->
 get_committed_update_seq(#db{committed_update_seq=Seq}) ->
     Seq.
 
-get_update_seq(#db{update_seq=Seq})->
-    Seq.
+get_update_seq(#db{} = Db)->
```

Comment:
This is a nit for me too but its historical and if I were to change it here the diff would spider out to all places that call these functions so I left them all as they were.